DAO/Classes/RealmDAO/DAO/RealmDAO.swift

```diff
@@ -271,6 +271,8 @@ open class RealmDAO<Model: Entity, RealmModel: RLMEntry>: DAO<Model> {
     
     
     private func cascadeDelete(_ object: AnyObject?) {
+        guard object?.isInvalidated == false else { return }
```

Comment:
```suggestion
        if object?.isInvalidated { return }
```

Reply:
@taflanidi @Hardenn It won't compile :) 'cause the value is optional.